internal/command/postgres/backup.go

```diff
@@ -459,10 +413,14 @@ func resolveRestoreTarget(ctx context.Context) string {
 	return target
 }
 
-func hasRequiredVersion(appName string, machines []*fly.Machine) error {
+func hasRequiredVersionForBackup(appName string, machines []*fly.Machine) error {
 	return hasRequiredVersionOnMachines(appName, machines, "", "0.0.53", "")
 }
 
+func hasRequiredVersionForBackupConfig(appName string, machines []*fly.Machine) error {
```

Comment:
This is fine, but unnecessary

Reply:
Why not? Do we not want to block on versions like v0.0.53 with backup support but not backup config support? Happy to remove it but the UX when someone runs `fly pg backup config` wouldn't clarify how they fix it.

Reply:
Oh, sorry, I should have clarified.  The version check is necessary, but abstracting it into a separate function isn't, since it's just issuing a return.